pkgs/top-level/python-packages.nix

```diff
@@ -10636,9 +10636,9 @@ self: super: with self; {
 
   proton-vpn-network-manager = callPackage ../development/python-modules/proton-vpn-network-manager { };
 
-  proton-vpn-network-manager-openvpn = callPackage ../development/python-modules/proton-vpn-network-manager-openvpn { };
+  proton-vpn-network-manager-openvpn = self.proton-vpn-network-manager; # Alias added 20-09-2024
 
-  proton-vpn-network-manager-wireguard = callPackage ../development/python-modules/proton-vpn-network-manager-wireguard { };
+  proton-vpn-network-manager-wireguard = self.proton-vpn-network-manager; # Alias added 20-09-2024
```

Comment:
Please use `python-aliases.nix` instead.

Reply:
I've converted the aliases into a throw error in python-aliases.nix since this is probably the better workflow in the long run.